README.md

```diff
@@ -251,85 +290,23 @@ To learn more see [Configuration
 Packages](https://crossplane.io/docs/v0.13/getting-started/package-infrastructure.html).
 
 ## What's Next
+
 The Crossplane community is targeting a v1.0 release with 90% coverage of all
 Cloud APIs by end of year 2020 with multiple workstreams in flight:
-1. Code gen of native Crossplane providers by adapting existing codegen pipelines:
-   - ACK Code Generation of the Crossplane `provider-aws`
-     - https://github.com/jaypipes/aws-controllers-k8s/tree/crossplane
-   - Azure Code Generation of the Crossplane `provider-azure`
-     - https://github.com/matthchr/k8s-infra/tree/crossplane-hacking
-2. Code gen of Crossplane providers that wrap the stateless Terraform providers
-   - Clouds that don't have code gen pipelines
-     - https://github.com/crossplane/crossplane/issues/262
+
+* Code gen of native Crossplane providers by adapting existing codegen pipelines:
```

Comment:
Was this meant to be converted to unordered list?

Reply:
sort of :). it's fails lint rules the way it was to mix ordered and unordered, and the ordered list for all of it didn't render OK in markdown, so I just switched it all to unordered for consistency and to work properly.